File: Codes in Classes/Class -01 ( 27-05-23 )/Codes/bubble_sort.py
# Sort an Array in Ascending Order (Bubble Sort)
n = int(input())
a = list(map(int, input().split()))

for i in range(n - 1):
    for j in range(i + 1, n):
        if a[i] > a[j]:
            a[i], a[j] = a[j], a[i]

for num in a:
    print(num, end=" ")

File: Codes in Classes/Class -01 ( 27-05-23 )/Codes/bubble_sort.cpp
#include <iostream>
using namespace std;

int main()
{
    // Sort an Array in Ascending Order (Bubble Sort)
    int n, tmp = 0;
    cin >> n;
    int a[n];

    for (int i = 0; i < n; i++)
    {
        cin >> a[i];
    }

    for (int i = 0; i < n - 1; i++)
    {
        for (int j = (i + 1); j < n; j++)
        {
            if (a[i] > a[j])
            {
                tmp = a[i];
                a[i] = a[j];
                a[j] = tmp;
            }
        }
    }

    for (int i = 0; i < n; i++)
    {
        cout << a[i] << " ";
    }

    return 0;
}
